minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/Client/MalmoModClient.java

```diff
@@ -243,20 +245,8 @@ public void onRightClickEvent(PlayerInteractEvent.RightClickBlock event){
         }
     }
 
-    /**
-     * Event listener that logs when agents open gui windows
-     * @param event the captured event
-     */
-    @SideOnly(Side.CLIENT)
```

Comment:
This was originally here to debug if we were opening a gui window unintentionally, probably OK to remove